lib/elixir/lib/agent.ex

```diff
@@ -105,7 +105,7 @@ defmodule Agent do
   The agent's state will be added to the given list as the first argument.
   """
 
-  @typedoc "Return values of `start*` functions"
+  @typedoc "Return values of _start*_ functions"
```

Comment:
It is fine how it was before. We don't use `_` anywhere in the docs and I wouldn't like to introduce it for this particular example.